docs/google/calibration.md

```diff
@@ -0,0 +1,170 @@
+     # The metric_value is of the type list and can have multiple values     
+     metric_value = latest_calibration[metric_name][qubit_or_pair]
+     print(f'{qubit_or_pair} = {metric_value}')
+```
+
+Calibration metrics will also soon be available from the 
+[Google Cloud Platform Console](https://console.cloud.google.com).
+
+## Individual Metrics
+
+Each metric can be referenced by its key in the calibration object, e.g.
+```latest_calibration['single_qubit_idle_t1_micros']```.
+
+Note that the metric names below are subject to change without notice.
+
+### P_00 readout error
+
+*   Metric key: single_qubit_readout_p0_error
+
+The p00 readout error is the probability that a qubit is incorrectly measured as
+|1> after being prepared in the |0> state.  This is mostly a measure of the
+error in measurement (read out) of the qubit while in the ground state,
+though this probability also contains the error than the qubit was not reset
+into the |0> ground state properly.  This is often called the SPAM (state 
+preparation and measurement) error.
+
+### P_11 readout error
+*   Metric key: single_qubit_readout_p1_error
+
+The P1 readout error is the probability that a qubit is incorrectly measured as
+|0> after being prepared in the |1> state.  This is dominated by the error in
+measurement (read out) of the qubit, but it implicitly contains several
+different types of error.  Also possible is that the excited state |1> was not
+prepared correctly or that the state decayed before measurement.  This error is
+generally expected to be higher than the P0 error.
+
+## Readout separation error
+*   Metric key: single_qubit_readout_separation_error
+
+When measured by the system, the |0> and |1> states manifest as outgoing analog
+signals.  These signals must be interpreted as being in one state or the other.
+Since these analog signals are continuous distributions, there will be some
+statistical overlap in the two distributions that would be theoretically
+impossible to distinguish.  This is classified as the separation error, and is
+calculated by fitting Gaussian distributions to the signals prepared in the
+|0> state and |1> state and calculating the overlap between the two
+distributions.
+
+## Isolated 1 qubit randomized benchmark error: 
+*   Metric key: single_qubit_rb_total_error
+*   Metric key: single_qubit_rb_purity_error
+
+Single qubit gate error is estimated using randomized benchmarking by taking
+sequences of varying length of the 24 gates within the Clifford group
+(those generated by the Pauli gates {±X,±Y,±Z}) then applying the inverse of the
+expected final state.  The result of this sequence should always be identity
+(|0> state).  The final error is measured and compared against this state to
+produce the total error.  This error is calculated for one qubit at a time while
+all other qubits on the device are idle (isolated).
+
+The purity error is a measure of the incoherent error (as opposed to
```

Comment:
I think people generally just speak of "purity" when they mean the trace of the square of the density matrix. Is this the quantity we report? If so, we should drop "error". If not, we should explain what it is we actually report.

Reply:
This metric is 1-tr(p^2), so one minus purity.  Elaborated this in the docs.

Reply:
Assuming that by p you mean the density matrix, this is then the (unnormalized) [linear entropy](https://en.wikipedia.org/wiki/Linear_entropy). It'd be good to use the established terminology.

Note: I'd indicate explicitly whether it is normalized or not since both are used. Normalized linear entropy is given by the formula (1 - tr(p^2)) * 2^n / (2^n - 1) and has values in [0, 1]. Unnormalized linear entropy is given by the formula 1 - tr(p^2) and has values in [0, 1 - 2^-n].

Reply:
Ok, I am not actually sure.  I will go find out before I write something incorrect.

Reply:
Thanks! Please ping this PR when it's ready for another look.

Reply:
I added a separate section on total vs purity error so I could elaborate further.  Can you review this and make sure I didn't say anything incorrect?

Reply:
Adjusted this section to include average vs pauli vs purity error.  Can you reevaluate to make sure I didn't say anything stupid?